apt/method.go

```diff
@@ -213,21 +235,20 @@ func (m *Method) handleConfigure(msg *Message) {
 		return
 	}
 	for _, configItem := range configs {
-		if strings.Contains(configItem, "Acquire::gar::Service-Account-JSON") {
-			parts := strings.SplitN(configItem, "=", 2)
-			if len(parts) != 2 {
-				// TODO: log this?
-				return
-			}
-			m.config.serviceAccountJSON = strings.TrimSpace(parts[1])
+		parts := strings.SplitN(configItem, "=", 2)
+		if len(parts) != 2 {
+			// TODO: log this?
+			return
 		}
-		if strings.Contains(configItem, "Acquire::gar::Service-Account-Email") {
-			parts := strings.SplitN(configItem, "=", 2)
-			if len(parts) != 2 {
-				// TODO: log this?
-				return
-			}
+		switch parts[0] {
+		case "Acquire::gar::Service-Account-JSON":
+			m.config.serviceAccountJSON = strings.TrimSpace(parts[1])
+		case "Acquire::gar::Service-Account-Email":
 			m.config.serviceAccountEmail = strings.TrimSpace(parts[1])
+		case "Debug::Acquire::gar":
+			if strings.TrimSpace(parts[1]) == "1" {
```

Comment:
I propose that this check for "true" and "yes" as well; maybe doing so in a case-insensitive way. Really, it could check for any value that isn't "0" or "false" or "False" or "no", too, if preferred.